Formula/keptn.rb

```diff
@@ -0,0 +1,22 @@
+class Keptn < Formula
+  desc "Is the CLI for keptn.sh a message-driven control-plane for application delivery"
+  homepage "https://keptn.sh"
+  url "https://github.com/keptn/keptn/archive/0.7.3.tar.gz"
+  sha256 "7c5ab4bce7f8c75371a6130ae7929a9c7a88d05f29b96604b3849a93d2177228"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    Dir.chdir("cli")
+    system "go", "build", "-ldflags", "-s -w -X main.Version=#{version} -X main.KubeServerVersionConstraints=\"\"", "-trimpath", "-o", bin/"keptn"
+  end
+
+  test do
+    run_output = shell_output("#{bin}/keptn version 2>&1")
+    assert_match "\nKeptn CLI version:", run_output
+
+    version_output = shell_output("#{bin}/keptn version 2>&1")
+    assert_match version.to_s, version_output
```

Comment:
we need some functional test than version test, you can x-check with the other go formulae.

Reply:
@chenrui333 Done. If something missing let me know :neckbeard: